src/CMakeLists.txt

```diff
@@ -145,16 +145,16 @@ if(WIN32)
 	)
 endif()
 
-if(POSIX)
```

Comment:
Actually, projects like midipix aim to provide POSIX syscalls on Windows using the native NT APIs.

The right thing to do here is to have actual checks for that.

Reply:
The `POSIX` variable is only true if `_POSIX_VERSION` is defined in `unistd.h`. If this change was actually necessary for you to produce a working build, then there's a good chance your toolchain is broken in some subtle way (maybe looking for system headers when cross-compiling?)

@lachs0r, is it not reasonable to assume that POSIX syscalls are available if we have `_POSIX_VERSION`?